test/sequential/test-net-connect-local-error.js

```diff
@@ -4,24 +4,16 @@ const assert = require('assert');
 const net = require('net');
 
 const client = net.connect({
-  port: common.PORT + 1,
-  localPort: common.PORT,
+  port: common.PORT,
+  localPort: common.PORT + 1,
   localAddress: common.localhostIPv4
 });
 
 client.on('error', common.mustCall(function onError(err) {
   assert.strictEqual(err.syscall, 'connect');
   assert.strictEqual(err.code, 'ECONNREFUSED');
```

Comment:
@sebastianplesciuc IMHO @santigimeno it right, we should just add an if for `EADDRINUSE` (or `err.code in ['ECONNREFUSED', 'EADDRINUSE']`)
/cc @Trott @cjihrig